tests/IntegrationTests/Notifications/Repositories/YoutubeRepositoryTests.cs

```diff
@@ -0,0 +1,606 @@
+		{
+			
+			// arrange
+			
+			var repository = new YoutubeRepository();
+
+			using var context = new ArkadiaDbContext();
+
+			var items = new YoutubeSubscription[]
+			{
+				new YoutubeSubscription
+				{
+					Id = "1",
+					ChannelTitle = "cooltitle",
+					ExpiresAt = DateTime.UtcNow.AddDays(10),
+					GuildIds = new[] { "guild1" }
+				},
+				new YoutubeSubscription
+				{
+					Id = "2",
+					ChannelTitle = "coolertitle",
+					ExpiresAt = DateTime.UtcNow.AddDays(3),
+					GuildIds = new[] { "guild1" }
+				}
+			};
+			
+			// act
+
+			foreach (var sub in items)
+			{
+				await repository.AddSubscriptionAsync(sub.Id, sub.ExpiresAt, sub.GuildIds[0], sub.ChannelTitle);
+			}
+
+			var subscriptions = repository.GetSubscriptions().ToArray();
+
+			// assert
+			
+			Assert.That(subscriptions, Is.Not.Empty);
+			Assert.That(subscriptions, Is.EquivalentTo(items).Using(new YoutubeSubscriptionComparer()));
+		}
+
+		[Test]
+		public async Task YoutubeRepository_AddSubscription_AddsSubscriptionCorrectly()
+		{
+			
+			// arrange
+
+			await using var repo = new YoutubeRepository();
+
+			const string id = "test";
+			const string channelTitle = "deez";
+			var expiryTime = DateTime.UtcNow.AddMinutes(100);
+			var guildId = "541738403230777351";
+
+			var expected = new YoutubeSubscription
+			{
+				Id = id,
+				ChannelTitle = channelTitle,
+				ExpiresAt = expiryTime,
+				GuildIds = new []{ guildId }
+			};
+			
+			// act
+
+			await repo.AddSubscriptionAsync(id, expiryTime, guildId, channelTitle);
+
+			var result = repo.GetSubscriptions().ToArray();
+			
+			// assert
+			
+			Assert.That(result, Has.Exactly(1).Items);
+			Assert.That(result.First(), Is.EqualTo(expected).Using(new YoutubeSubscriptionComparer()));
+		}
+		
+		[Test]
+		public async Task YoutubeRepository_GetGuildByIdOrDefaultAsync_ReturnsNull_WhenEntryDoesNotExist()
+		{
+			
+			// arrange
+
+			var repository = new YoutubeRepository();
+			const string name = "test";
+
+			// act
+
+			var response = await repository.GetGuildByIdOrDefaultAsync(name);
+
+			// assert
+			
+			Assert.That(response, Is.Null);
+		}
+
+		[Test]
+		public async Task YoutubeRepository_GetGuildByIdOrDefaultAsync_ReturnsItem_WhenExists()
+		{
+			// arrange
+
+			var repository = new YoutubeRepository();
+
+
+			await using var context = new ArkadiaDbContext();
+
+			var guild = CreateDefaultGuild();
+
+			// act
+			
+			await context.Guilds.AddAsync(guild);
+
+			await context.SaveChangesAsync();
+
+			var response = await repository.GetGuildByIdOrDefaultAsync(guild.Id);
+
+			// assert
+
+			Assert.That(response, Is.Not.Null);
+			Assert.That(response!.Id, Is.EqualTo(guild.Id));
+			Assert.That(response.YoutubeUploadLiveChannel, Is.EqualTo(guild.YoutubeUploadLiveChannel));
+		}
+
+		[Test]
+		public async Task YoutubeRepository_UpsertGuildAsync_UpsertsCorrectly_WhenGuildIsNull()
+		{
+			// arrange
+
+            var repository = new YoutubeRepository();
```

Comment:
Why is this space-indented? Did you run format?

Reply:
Also not the only file with space indentation, JFYI. Might want to run the formatter on the entire solution.